hermes/bin/Cargo.toml

```diff
@@ -63,12 +62,18 @@ console = { workspace = true }
 serde = { workspace = true, features = ["derive"] }
 serde_json = { workspace = true }
 jsonschema = { workspace = true }
+hex = { workspace = true }
+hmac = { workspace = true }
+pbkdf2 = { workspace = true }
+sha2 = { workspace = true }
+ed25519 = { workspace = true }
```

Comment:
should ed25519-dalek be sufficient ? What's in ed25519?

Reply:
Yea, good catch, `ed25519` unused dep, forgot to remove it